arm-sql/2015-05-01/swagger/sql.json

```diff
@@ -0,0 +1,227 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "SqlManagementClient",
+        "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete Databases, Servers and related resources.",
+        "version": "2015-05-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json",
+        "text/json"
+    ],
+    "produces": [
+        "application/json",
+        "text/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/securityAlertPolicies/Default": {
+            "put": {
+                "tags": [
+                    "SecurityAlertPolicy"
+                ],
+                "operationId": "SecurityAlertPolicy_CreateOrUpdateServerSecurityAlertPolicy",
```

Comment:
I would recommend having the name like this `SecurityAlertPolicy_CreateOrUpdateServer`. The reason being, generated client will have operation group of the name `SecurityAlertPolicy` (the part before underscore) and that operation group will have a method named `CreateOrUpdateServer`. Having the phrase SecurityAlertPolicy would make it longer and redundant.
IMO, `client.SecurityAlertPolicy.CreateOrUpdateServer()` looks crisp v/s `client.SecurityAlertPolicy.CreateOrUpdateServerSecurityAlertPolicy()`.